backend/app/assets/javascripts/spree/backend/views/tables/selectable_table_sum_item_amount.js

```diff
@@ -0,0 +1,32 @@
+Spree.Views.Tables.SelectableTable.SumItemAmount = Backbone.View.extend({
+  initialize: function(options) {
+    this.listenTo(this.model, 'change', this.render)
+
+    this.render();
+  },
+
+  render: function() {
+    var html = HandlebarsTemplates['tables/return_item_sum_amount']({
```

Comment:
This file is both generic (it's called with a generic `SumItemAmount` name) and specific since it only works with this return items table. I'd say we can improve it by passing dynamic data (like the template and the label to use) from outside. Do you think it's doable?

Reply:
I refactored the code. I think we can add the ability to customize the method to recalculate the item amount.
Right now the commit creates a specific object to wrap the generic object and it provides to the generic object these data.